ddtrace/profiling/profiler.py

```diff
@@ -43,27 +42,18 @@ class Profiler(object):
     def __init__(self, *args, **kwargs):
         self._profiler = _ProfilerInstance(*args, **kwargs)
 
-    def start(self, stop_on_exit=True, profile_children=True):
-        """Start the profiler.
-
-        :param stop_on_exit: Whether to stop the profiler and flush the profile on exit.
-        :param profile_children: Whether to start a profiler in child processes.
-        """
+    def start(self):
+        """Start the profiler."""
 
-        if profile_children:
-            try:
-                uwsgi.check_uwsgi(self._restart_on_fork, atexit=self.stop if stop_on_exit else None)
-            except uwsgi.uWSGIMasterProcess:
-                # Do nothing, the start() method will be called in each worker subprocess
-                return
+        try:
+            uwsgi.check_uwsgi(self._restart_on_fork, atexit=self.stop)
+        except uwsgi.uWSGIMasterProcess:
+            # Do nothing, the start() method will be called in each worker subprocess
+            return
 
         self._profiler.start()
 
-        if stop_on_exit:
-            atexit.register(self.stop)
-
-        if profile_children:
-            forksafe.register(self._restart_on_fork)
```

Comment:
I cannot understand why the profile tests are failing 🙁 . I have tried putting this line back, but it doesn't seem to solve the problem. I'd expect the data to be flushed anyway at process shutdown. @taegyunkim any help would be greatly appreciated 🙏